### ISSUE TITLE
No source maps

### ISSUE THREAD

**Issue by davidknezic** [milankinen/livereactload#106]
Seems that the plugin doesn't support source maps.

Running it like this doesn't include any source maps into my build:

``` sh
watchify index.js -d -t babelify -g envify -p livereactload -o bundle.js
```

However, compiling my bundle without the plugin gets me all the source maps inlined:

``` sh
watchify index.js -d -t babelify -g envify -o bundle.js
```

Is there a way to add that feature?

**Comment by milankinen** [comment]
Hmm.. Have you tried to use inlined source maps? I've got them working in my projects.

**Comment by milankinen** [comment]
Closing this one since there is not activity lately.

**Comment by iamjochem** [comment]
hi @milankinen - this is still an issue, please reopen.

I'm currently using **watchify v3.7.0** & **livereactload 2.2.4** - when running watchify without `-p livereactload` inlined sourcemaps are included in the generated bundle, with `-p livereactload` the sourcemap is missing from the bundle.

In my case I am using babelify transform but specifying it via the `browserify` property in `package.json`, e.g.:

```js
  "browserify": {
    "transform": [
      "babelify",
      "brfs"
    ]
  },
```

P.S. I have tried with livereactload 2.2.1 through 2.2.3 - just in case it was a regression in the last couple of versions ... they all exhibit the same behaviour of stripping the sourcemap from the browserified bundle.

**Comment by milankinen** [comment]
Thanks! Re-opened the issue. Could you test the new (beta) version `livereactload@2.3.0-rc7` and tell whether the source maps works in your case or not?

**Comment by iamjochem** [comment]
hi there, just gave it a shot ... still no source map in the output, I'm piping through `exorcist` so I basically get the following warning:

> The code that you piped into exorcist contains no source map!
Therefore it was piped through as is and no external map file generated.

my full command line looks like so:

```sh
./node_modules/.bin/watchify \
-v --debug --delay=100 --fast --dg=0 -t babelify -p livereactload \
   /my/project/client-entry-point.js \
  -o './node_modules/.bin/exorcist /my/project/build/public/js/bundle.js.map > /my/project/build/public/js/bundle.js'
```

I have a `.babelrc` file with the following in it (note that the `sourceMaps` property is only in there because I'm tried to get this to work... I don't _think_ I actually need it):

```json
{
    "presets"               : [
        "es2015",
        "react"
    ],
    "compact"               : true,
    "minified"              : false,
    "comments"              : true,
    "sourceMaps"            : true,

    "env"                   : {
        "development"       : {
            "plugins"       : [
                [ "react-transform", { "transforms": [ { "transform": "livereactload/babel-transform", "imports": ["react"] } ] } ]
            ]
        }
    }
}
```

**Comment by milankinen** [comment]
Should be fixed now in  [`3.1.1`](https://github.com/milankinen/livereactload/releases/tag/3.1.1)
